src/docfx/config/Config.cs

```diff
@@ -10,8 +10,13 @@ namespace Microsoft.Docs.Build
 {
     internal sealed class Config
     {
-        private static readonly string[] s_defaultInclude = new[] { "docs/**/*.{md,yml,json}" };
-        private static readonly string[] s_defaultExclude = new[] { "_site/**/*", "localization/**/*" };
+        private static readonly string[] s_defaultInclude = new[] { "**/*.{md,yml,json}" };
+        private static readonly string[] s_defaultExclude = new[]
+        {
+            "_site/**",             // Default output location
```

Comment:
Why is `_site/**` working the same as `_site/**/*`?

Reply:
`**` matches zero or more directories, we are do string comparison, so it literally means everything in that sub-directory. This behavior is the same as [node-glob](https://github.com/isaacs/node-glob)